client/app/components/conference_room/provider/EventProvider.jsx

```diff
@@ -34,6 +37,38 @@ export default class EventProvider extends React.Component {
     this._fetchForToday();
   }
 
+  handleConfirm() {
```

Comment:
Is there anything bad that can happen when `handleConfirm` / `handleFinish` / `handleCreate` are called multiple times?